change/@fluentui-react-combobox-e478d812-6a6a-4b46-88a4-6a97b5dbcfbd.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "fix: \"update disabled + underline styles, have all text-like form control disabled states match",
```

Comment:
```suggestion
  "comment": "fix: update disabled + underline styles, have all text-like form control disabled states match",
```

Reply:
oops, this is what I get for trying to copy/paste in WSL 😅. Thanks for catching it!